spec/factories.rb

```diff
@@ -33,9 +33,14 @@
     phone "999-999-9999"
 
     trait :confirmed do
-      confirmed_at { Time.zone.now }
+      confirmed_at  Time.zone.now
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.